OpenSim/Common/C3DFileAdapter.cpp

```diff
@@ -69,13 +69,19 @@ C3DFileAdapter::extendRead(const std::string& fileName) const {
     reader->Update();
     auto acquisition = reader->GetOutput();
 
+    EventTable event_table{};
+    auto events = acquisition->GetEvents();
+    for (auto it = events->Begin();
+        it != events->End();
+        ++it) {
+        auto et = *it;
+        event_table.push_back({ et->GetLabel(),
+            et->GetTime(),
+            et->GetFrame(),
+            et->GetDescription() });
+    }
+
```

Comment:
I thought it read better to get the events first, which are then added to both the marker and force tables that are created in the two code blocks below.  I also thought I might be able to extract the start time for the acquisition at some point.